ci/infrastructure/scripts/infrastructure.sh

```
@@ -262,6 +264,7 @@ defaultSettings() {
   VS_DATESTAMP=$(date +%Y%m%d)
   VS_HOST_IP_ADDRESS=$(/usr/sbin/ip ad sh  | egrep "global noprefixroute" | awk '{print $2}' | sed -e "s/\/.*$//")
   VS_PARENT_JOB_NAME=$(echo $JOB_NAME | sed -e "s/\/.*//g")
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)